### ISSUE TITLE
"Scene Entity" Node should allow non-overridden children to be picked

### ISSUE THREAD

**Issue by leonardo-ourmachinery** [OurMachinery/themachinery-public#1188]
![image](https://user-images.githubusercontent.com/70660839/158983655-680338d6-320b-4998-86d0-2c0709e467c6.png)
![image](https://user-images.githubusercontent.com/70660839/158983542-f43835c0-a73c-4e9d-b8da-9003783dc875.png)

 In this situation it would be nice to be able to pick the "Camera Pivot" child of the xbot entity in the "Scene Entity" node without having to override it.

**Comment by simon-ourmachinery** [comment]
I think thats not an enhancement thats a bug.

**Comment by niklas-ourmachinery** [comment]
I agree that would be nice, but there is no easy way to do that, since the *Scene Entity* node picks a `tm_tt_id_t` and inherited items are not guaranteed to have a unique `tm_tt_id_t`.

I think for this you would have to use a *Scene Entity From Path* node (or similar) -- that way you could specify the path as `xbot/CameraPivot` -- it would be nice if that node autocompleted too.

Also, I think the "enhancement" is the right label, since it's about adding support for picking an item that we can't pick today.
